app/styles/partials/footer.scss

```diff
@@ -39,3 +39,7 @@ footer {
     right: 0 !important;
   }
 }
+
+  .ui.card > .content {
+  padding: 0.35rem;
+  }
```

Comment:
This will add padding to all cards, not just event cards

Reply:
so i need to fix it only in event cards.